src/App.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * App
+ * Define a Restful and a SOAP Webservice and routes incoming requests to controllers
+ */
+
+import * as bodyParser from "body-parser";
+import * as express from "express";
+import * as core from "express-serve-static-core";
+import * as http from "http";
+import { reporters } from "italia-ts-commons";
+import { CONFIG, Configuration } from "./Configuration";
+import * as PaymentController from "./controllers/restful/PaymentController";
+import { logger } from "./utils/Logger";
+
+// Define and start a WS SOAP\Restful Server
+export function startApp(): http.Server {
```

Comment:
I'd rather move all the code inside startApp() into `index.ts` at module top level getting rid of this function here and importing CONFIG from index.ts (not in this module which is supposed to be usable with different configurations)

Reply:
I think we could mode just the the configuration part outside:
const config = Configuration.decode(CONFIG).getOrElseL(errors => {
    throw new Error(
      `Invalid configuration: ${reporters.readableReport(errors)}`
    );
  });

startApp() is still useful to define an express server and put it into listening mode.

Reply:
that's ok (if I undestood well, then startApp will take config as input)

Reply:
Exactly